bigquery/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       # deployments; bootstrap checks are still enabled for production
       # deployments.
       - "transport.host=localhost"
-      - "ES_JAVA_OPTS=-Xms2g -Xmx2g"
+      - ES_JAVA_OPTS=${ES_JAVA_OPTS:--Xms256m -Xmx256m}
```

Comment:
This can just be
```
- ES_JAVA_OPTS
```
right?

After this change, please manually test with and without ES_JAVA_OPTS, just to confirm things work.

Reply:
Done.